sdk/communication/communication-short-codes/tests.yml

```diff
@@ -0,0 +1,20 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure-tools/communication-short-codes"
```

Comment:
Why is this package in azure-tools repo? Shouldn't this be @azure scope

Reply:
Hello Praveen! This SDK is not going to be publicly supported as there are no external customers now or the foreseeable future. The decision by the board was to publish it under this namespace.